### PR TITLE
Ajax request for checkin in quickicons

### PR DESCRIPTION
**Pull request by chmst**
Pull Request for Issue #95 .

### Summary of Changes
Some quickicons show the amount of enabled entries in the database. This requires database queries. In case of the checkin icon, all important tables have to be checked which results in duplicate queries is time consuming. 

In this PR an ajax request decouples database queries from the output of the screen. 

Should we use ajax for all icons? It looks strange if the screen shows only spinner icons at the first glance.

### CODE REVIEW

File: administrator/components/com_cpanel/tmpl/cpanel/default.php [hunk 1]
```diff
@@ -42,17 +42,15 @@
...
-<div class="row">
-	<div class="col-md-12">
-    	<?php
-		// Display the icon position modules
-		foreach ($this->quickicons as $iconmodule)
-		{
-			echo ModuleHelper::renderModule($iconmodule);
-		}
+	<div class="row">
+		<?php
```
Comment (brianteeman) [inline administrator/components/com_cpanel/tmpl/cpanel/default.php]: why is the col removed?

Comment (chmst) [reply administrator/components/com_cpanel/tmpl/cpanel/default.php]: My fault - small confusion on my repo,

File: build/media_source/mod_quickicons/js/quickicons.es6.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            onSuccess: (resp) => {
+              let response;
+              try {
+                response = JSON.parse(resp);
+              } catch (error) {
+                throw new Error('Failed to parse JSON');
+              }
+		if (response.data) {
+                const elem = document.createElement('span');
+                elem.innerHTML = response.data;
```
Comment (C-Lodder) [inline build/media_source/mod_quickicons/js/quickicons.es6.js:33]: Don't use `innerHTML` for security reasons

File: build/media_source/mod_quickicons/js/quickicons.es6.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+((document, Joomla) => {
+  'use strict';
+
+  const init = () => {
+    // Cleanup
+    document.removeEventListener('DOMContentLoaded', init);
+
+    // Get the elements
+    const elements = [].slice.call(document.querySelectorAll('.quickicon-counter'));
+console.log(elements);
```
Comment (C-Lodder) [inline build/media_source/mod_quickicons/js/quickicons.es6.js]: remove `console.log`

**Review by C-Lodder — CHANGES_REQUESTED** [review]

**Comment by chmst** [comment]
Closing this as a new PR follows

**Pull request closed without merging**